src/java.base/share/classes/jdk/internal/util/ClassFileDumper.java

```diff
@@ -79,10 +77,6 @@ public static ClassFileDumper getInstance(String key, String path) {
     private final AtomicInteger counter = new AtomicInteger();
 
     private ClassFileDumper(String key, String path) {
-        /*
```

Comment:
The comment might still be relevant if changed to System.getProperty.

Reply:
It's fine now: previously user defined security manager was effectively loading user code that uses java.lang.invoke, now users cannot specify security managers, so this problem does not exist any more.

Reply:
In that case, the code should be calling System.getProperty and not the backup VM saved props.

Reply:
@liach Does this mean we can/should use `System.getProperty` here?

Reply:
VM saved props is still better imo as user code cannot tamper it with System.setProperty.

Reply:
VM.save props should only be used if there is a bootstrap issue.
If tampering is a concern, then the property should be added to StaticProperties.

Reply:
The use of the properties in ClassFileDumper is all in support of debug capabilities and is not sensitive.

Reply:
Either way is fine.  I think we probably need to review the majority of uses of VM.savedPropoerty as they mostly relate to SM boot circularity.  So we will change this one either now or later.

Reply:
I would prefer if we could deal with getSavedProperty as a wider area follow up, since that would creep out of scope for this particular PR.

Ok with you, @liach ?

Perhaps @seanjmullan has input on how to deal with VM.getSavedProperty across the code base?

Reply:
I prefer to minimize code changes for these cleanup issues. We should file a separate issue, "Examine usages of VM.getSavedProperty after JEP 486" or something like that.

Reply:
Thanks Sean, I filed https://bugs.openjdk.org/browse/JDK-8344335 to track this.